model/src/main/proto/javaclasses/exlibris/q/values.proto

```diff
@@ -91,28 +88,27 @@ message LoanDetails {
     google.protobuf.Timestamp when_due = 4;
 }
 
-// Details about inventory.
+// Detailed information about the inventory.
 //
-// View consists of fields with book title, authors,
-// available books, amount of overdue books.
+// View consists of fields with book title, authors, book location.
 //
 message InventoryDetails {
 
-    // book title.
+    // Book title.
     BookTitle book_title = 1;
 
-    // todo : dima
+    // List of the author's full name.
     AuthorName author_names = 2;
 
-    // List with information about books.
+    // List of information about the location of books.
     repeated InventoryItem items = 3;
 }
 
-//
+// Information about the location of books.
 //
 message InventoryItem {
```

Comment:
I think that the message that has information about the location of the item, should consist it in its name.